src/loaders/resourceLoader.ts

```diff
@@ -129,9 +129,21 @@ export abstract class ResourceLoader implements IResourceBase {
     // eslint-disable-next-line @typescript-eslint/no-unused-vars
     forceRefresh: boolean = false,
   ): Promise<string[]> {
-    const schemaRegistry = await this.resolveSchemaRegistry(registryOrEnvironmentId);
-
-    return await fetchSubjects(schemaRegistry);
+    try {
+      const schemaRegistry = await this.resolveSchemaRegistry(registryOrEnvironmentId);
+      return await fetchSubjects(schemaRegistry);
+    } catch (error) {
+      logger.error("Error fetching subjects", error);
+      if (
+        error instanceof Error &&
+        error.message.match(/No schema registry found for environment/)
```

Comment:
Where does this error message come from?

Reply:
resolveSchemaRegistry(), line 173:

schemaRegistry = await this.getSchemaRegistryForEnvironmentId(registryOrEnvironmentId);
      if (!schemaRegistry) {
        throw new Error(`No schema registry found for environment ${registryOrEnvironmentId}`);
      }